README.md

```diff
@@ -34,16 +34,17 @@ dns:
   suffix: <domain name>
 
 ssh:
-  key: mykey
+  # Generate SSH key pair: e.g. `ssh-keygen -t rsa -b 4096 -C "me@here.com" -f openshift-key`
+  key: <name of the key pair, e.g., openshift-key>
 
 users:
   - username: admin
     password: password
     admin: true
-  - username: user
+  - username: user1
     password: password
     sudoer: true
-  - username: user
+  - username: user2
```

Comment:
I am OK with the `user1` ... but `user2` here does not make sense, as you are going to generate a range of usernames and user probably want to do `user0` over to `userXY` rather than `user20` over to `user2XY`